packages/admin-next/dashboard/src/v2-routes/promotions/promotion-create/components/create-promotion-form/create-promotion-form.tsx

```diff
@@ -0,0 +1,791 @@
+                    return (
+                      <Form.Item>
+                        <Form.Label>
+                          {t("promotions.fields.value_type")}
+                        </Form.Label>
+                        <Form.Control>
+                          <RadioGroup
+                            className="flex gap-y-3"
+                            {...field}
+                            onValueChange={field.onChange}
+                          >
+                            <RadioGroup.ChoiceBox
+                              value={"fixed"}
+                              label={t(
+                                "promotions.form.value_type.fixed.title"
+                              )}
+                              description={t(
+                                "promotions.form.value_type.fixed.description"
+                              )}
+                              className={clx("basis-1/2", {
+                                "border-2 border-ui-border-interactive":
+                                  "fixed" === field.value,
+                              })}
+                            />
+
+                            <RadioGroup.ChoiceBox
+                              value={"percentage"}
+                              label={t(
+                                "promotions.form.value_type.percentage.title"
+                              )}
+                              description={t(
+                                "promotions.form.value_type.percentage.description"
+                              )}
+                              className={clx("basis-1/2", {
+                                "border-2 border-ui-border-interactive":
+                                  "percentage" === field.value,
+                              })}
+                            />
+                          </RadioGroup>
+                        </Form.Control>
+                        <Form.ErrorMessage />
+                      </Form.Item>
+                    )
+                  }}
+                />
+
+                <div className="flex gap-y-4">
+                  <Form.Field
+                    control={form.control}
+                    name="application_method.value"
+                    render={({ field: { onChange, ...field } }) => {
+                      return (
+                        <Form.Item className="basis-1/2">
+                          <Form.Label>
+                            {isFixedValueType
+                              ? t("fields.amount")
+                              : t("fields.percentage")}
+                          </Form.Label>
+                          <Form.Control>
+                            {isFixedValueType ? (
+                              <CurrencyInput
+                                min={0}
+                                onValueChange={onChange}
+                                code={"USD"}
+                                symbol={getCurrencySymbol("USD")}
```

Comment:
Yes, you're right. This can't be done right now because we're accepting a multi select for currencies. There is a ticket in the followup that makes it just a singular currency, this will be changed when that is done.